<h1 class="devsite-page-title">tf.errors.UnauthenticatedError</h1>     <div itemscope itemtype="http://developers.google.com/ReferenceObject"> <meta itemprop="name" content="tf.errors.UnauthenticatedError"> <meta itemprop="path" content="Stable"> <meta itemprop="property" content="error_code"> <meta itemprop="property" content="message"> <meta itemprop="property" content="node_def"> <meta itemprop="property" content="op"> <meta itemprop="property" content="__init__"> </div>  <table class="tfo-notebook-buttons tfo-api" align="left"> <td> <a target="_blank" href="https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/errors/UnauthenticatedError">  TensorFlow 1 version</a> </td> <td> <a target="_blank" href="https://github.com/tensorflow/tensorflow/blob/r2.1/tensorflow/python/framework/errors_impl.py#L339-L350">  View source on GitHub </a> </td>
</table>  <h2 id="class_unauthenticatederror">Class <code translate="no" dir="ltr">UnauthenticatedError</code>
</h2> <p>The request does not have valid authentication credentials.</p> <p>Inherits From: <a href="operror"><code translate="no" dir="ltr">OpError</code></a></p>  <p>This exception is not currently used.</p> <h2 id="__init__"><code translate="no" dir="ltr">__init__</code></h2> <p><a target="_blank" href="https://github.com/tensorflow/tensorflow/blob/r2.1/tensorflow/python/framework/errors_impl.py#L347-L350">View source</a></p> <pre class="prettyprint lang-python" translate="no" dir="ltr" data-language="python">__init__(
    node_def,
    op,
    message
)
</pre> <p>Creates an <code translate="no" dir="ltr">UnauthenticatedError</code>.</p> <h2 id="properties">Properties</h2> <h3 id="error_code"><code translate="no" dir="ltr">error_code</code></h3> <p>The integer error code that describes the error.</p> <h3 id="message"><code translate="no" dir="ltr">message</code></h3> <p>The error message that describes the error.</p> <h3 id="node_def"><code translate="no" dir="ltr">node_def</code></h3> <p>The <code translate="no" dir="ltr">NodeDef</code> proto representing the op that failed.</p> <h3 id="op"><code translate="no" dir="ltr">op</code></h3> <p>The operation that failed, if known.</p> <p><em>N.B.</em> If the failed op was synthesized at runtime, e.g. a <code translate="no" dir="ltr">Send</code> or <code translate="no" dir="ltr">Recv</code> op, there will be no corresponding <a href="../operation"><code translate="no" dir="ltr">tf.Operation</code></a> object. In that case, this will return <code translate="no" dir="ltr">None</code>, and you should instead use the <a href="operror#node_def"><code translate="no" dir="ltr">tf.errors.OpError.node_def</code></a> to discover information about the op.</p> <h4 id="returns">Returns:</h4> <p>The <code translate="no" dir="ltr">Operation</code> that failed, or None.</p> <h2 id="compat_aliases">Compat aliases</h2> <ul> <li><a href="unauthenticatederror"><code translate="no" dir="ltr">tf.compat.v1.errors.UnauthenticatedError</code></a></li> <li><a href="unauthenticatederror"><code translate="no" dir="ltr">tf.compat.v2.errors.UnauthenticatedError</code></a></li> </ul>  <devsite-page-rating position="footer" selected-rating="0" hover-rating-star="0"> </devsite-page-rating><div class="_attribution">
  <p class="_attribution-p">
    &copy; 2019 The TensorFlow Authors. All rights reserved.<br>Licensed under the Creative Commons Attribution License 3.0.<br>Code samples licensed under the Apache 2.0 License.<br>
    <a href="https://www.tensorflow.org/api_docs/python/tf/errors/UnauthenticatedError" class="_attribution-link">https://www.tensorflow.org/api_docs/python/tf/errors/UnauthenticatedError</a>
  </p>
</div>
